tests/discovery/test_openreview.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from pytest_regressions.file_regression import FileRegressionFixture
+
+from paperoni import config
+from paperoni.discovery import openreview
+from paperoni.discovery.base import PaperInfo
+from paperoni.discovery.openreview import OpenReview, OpenReviewDispatch
+from paperoni.model.classes import Paper
+
+from ..utils import check_papers, iter_links_ids, iter_releases
+
+
+@pytest.mark.parametrize(
+    "query_params",
+    [
+        {"venue": "NeurIPS.cc/2020/Conference"},
+        {"venue": "NeurIPS.cc/2024/Conference"},
+        {"paper_id": "gVTkMsaaGI", "venue": "NeurIPS.cc/2024/Conference"},
+        {"author": "Yoshua Bengio", "venue": "NeurIPS.cc/2024/Conference"},
+        {"author_id": "~Yoshua_Bengio1", "venue": "NeurIPS.cc/2024/Conference"},
```

Comment:
I think author_id is sufficient here. venue is only needed for non-id search fields.